cli/hzn.go

```diff
@@ -45,9 +46,13 @@ Environment Variables:
       to specify the organization ID'.
   HZN_EXCHANGE_USER_AUTH:  Default value for the 'hzn exchange -u' or 'hzn
       register -u' flag, in the form '[org/]user:pw'.
-  HZN_DONT_SUBST_ENV_VARS:  Set this to "1" to indicate that input json files
```

Comment:
Is the idea behind removing this from the help that we will still support it as a hidden option, but almost no one will need to know about it, so don't bother confusing them with it?

Reply:
You are absolutely right.